src/containers/ConversionForm/index.tsx

```diff
@@ -109,6 +113,8 @@ const ConversionForm = ({
     const cancelProcessing = () => {
         setIsProcessing(false);
         setConversionStatus({ status: CONVERSION_NO_SERVER });
+        // todo - keep old trajectory and timestamp when cancelling conversion request
+        clearSimulariumFile({ newFile: false });
```

Comment:
There are some issues currently in Octopus with returning to the previous trajectory once conversion has started, and Blair has requested that for now, once a conversion request is initiated, we clear out the viewer.